Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -258,14 +261,21 @@ public class HTTPServer: Server {
 
      ### Usage Example: ###
      ````swift
-     try server.listen(on: 8080)
+     try server.listen(on: 8080, node: "localhost")
      ````
 
      - Parameter on: Port number for new connections, e.g. 8080
-    */
-    public func listen(on port: Int) throws {
+     - Parameter node: has the same meaning as in `getaddrinfo()`
+     */
+    public func listen(on port: Int, node: String?) throws {
         self.port = port
-        try listen(.tcp(port))
+        self.node = node
+        try listen(.tcp(port, node))
+    }
+
+    @available(*, deprecated, message: "use 'listen(on:node) throws' instead")
```

Comment:
I'm not sure if we should deprecate `listen(on:)`. We could make this a simple additive change by introducing a new `listen(on:node)` or `listen(on:address)`